editor/Dockerfile

```diff
@@ -88,7 +93,7 @@ RUN [ `echo $version-$module | grep -v '^\(2018.3\|2018.4\).*-android'` ] && exi
   && apt-get update -qq \
   && apt-get install -qq -y --no-install-recommends unzip \
   && mkdir -p ${ANDROID_SDK_ROOT} \
-  && chown -R 777 ${ANDROID_INSTALL_LOCATION} \
+  && chmod -R 777 ${ANDROID_INSTALL_LOCATION} \
```

Comment:
Was this in fact giving an error? If not, it would be better to make separate PRs for changes that are not related to supporting multiple modules.

Reply:
I didn't notice the error, but I've noticed this when adding that chmod for sdkmanager.

Reply:
separated into https://github.com/game-ci/docker/pull/119